service/src/main/java/com/carhouse/service/CarMakeService.java

```diff
@@ -1,14 +1,16 @@
 package com.carhouse.service;
 
 import com.carhouse.model.CarMake;
+import javassist.NotFoundException;
```

Comment:
`javassist` package?
Are you sure?

Reply:
you told me to find a ready-made NotFoundException class
this is it